signer/src/storage/mod.rs

```diff
@@ -271,6 +271,13 @@ pub trait DbRead {
         chain_tip: &model::BitcoinBlockHash,
         context_window: u16,
     ) -> impl Future<Output = Result<Option<model::SweepTransaction>, Error>> + Send;
+
+    /// Get the sweep transaction package for the given previous transaction id
+    /// (i.e. the previous output that was spent from the then-UTXO).
```

Comment:
I would add a little here and maybe rephrase it. I would say something like "This function takes the transaction ID of the signers' unspent output on the canonical Bitcoin blockchain and returns the chain of mempool transactions that spend that output." And maybe add something like this "The provided txid refers to an on-chain unspent output created by the signers, which may be spent by a transaction currently in the mempool."

Man that was wordy.

Reply:
Updated the comments a bit in [0d4aed9](https://github.com/stacks-network/sbtc/pull/810/commits/0d4aed96c6ec3706a382ee98797298178da90c9f).

While what you describe is the _intended use_, it's not really true for the provided functionality of the method. It purposely places the burden of making sure you're providing the correct `prevout_txid` on the caller and thus itself provides no guarantees of canonicality or that the `prevout_txid` is indeed confirmed.

Reply:
Actually renamed the method in [413ae4c](https://github.com/stacks-network/sbtc/pull/810/commits/413ae4ce3d7195a326e8a9695866d605d5ec9963) to better reflect what it does